pkgs/tools/networking/network-manager/fortisslvpn/default.nix

```diff
@@ -28,6 +28,7 @@ in stdenv.mkDerivation rec {
     "--without-libnm-glib"
     "--with-gnome=${if withGnome then "yes" else "no"}"
     "--localstatedir=/tmp"
+    "--enable-absolute-paths=yes"
```

Comment:
No need to specify yes, it is the default for `--enable-foo` options

Reply:
fixed